apps/astarte_appengine_api/lib/astarte_appengine_api/rooms/room.ex

```diff
@@ -200,8 +200,8 @@ defmodule Astarte.AppEngine.API.Rooms.Room do
       room_cleanup(state)
 
       :telemetry.execute(
-        [:astarte, :appengine, :channels, :room_closed],
-        %{},
+        [:astarte, :appengine, :channels, :room_opened],
```

Comment:
I see the point of doing this (summing the `-1` with the same event name) but I'm not sure if emitting a `:room_opened` event when the room is closed is the correct behaviour.
Did you try to look at other libraries that do similar stuff and see how they do it?